File: examples/ts/list-projects.ts
import { BisibilityClient } from "@bisibility/sdk";

const apiKey = process.env.BISIBILITY_API_KEY;
const baseUrl = process.env.BISIBILITY_BASE_URL;

if (!apiKey || !baseUrl) {
  throw new Error("Set BISIBILITY_API_KEY and BISIBILITY_BASE_URL before running this example.");
}

const bisibility = new BisibilityClient({
  apiKey,
  baseUrl,
});

const projects = await bisibility.projects.list();

for (const project of projects.data) {
  console.log(project.id, project.name);
}

console.log("OK ts-list-projects");

File: examples/python/list_projects.py
import os

from bisibility import BisibilityClient

api_key = os.environ["BISIBILITY_API_KEY"]
base_url = os.environ["BISIBILITY_BASE_URL"]

with BisibilityClient(api_key=api_key, base_url=base_url) as client:
    projects = client.list_projects()

    for project in projects.data:
        print(project.id, project.name)

print("OK python-list-projects")
